Examples/Modules/qed/breit_wheeler/inputs_3d_optical_depth_evolution

```diff
@@ -185,10 +195,10 @@ qed_bw.lookup_table_mode = "dummy_builtin"
 
 ### EXTERNAL E FIELD ### (3e15 * [-0.811107105653813 0.324442842261525 0.486664263392288] )
 particles.E_ext_particle_init_style = "constant"
-particles.E_external_particle = -2433321316961438 973328526784575 1459992790176863
+particles.E_external_particle = -486664263392286 194665705356914 291998558035372
 ####
 
 ### EXTERNAL B FIELD ### (1e7 * [0.28571429 0.42857143 0.85714286] )
 particles.B_ext_particle_init_style = "constant"
-particles.B_external_particle = 2857142.85714286 4285714.28571428 8571428.57142857
+particles.B_external_particle = 571429.7142856 857142.5714284 1714285.142857
```

Comment:
I am not sure to understand this change. Could you please explain why you would like to change the field intensities here?

Reply:
(ah, sorry... I failed to notice that the PR is still `[WIP]`)